ngraph_bridge/ngraph_utils.h

```diff
@@ -329,6 +329,16 @@ Status CheckAxisDimInRange(std::vector<int64> axes, size_t rank);
 Status NgraphSerialize(const std::string&,
                        const std::shared_ptr<ngraph::Function>&);
 
+#if (CMAKE_BUILD_TYPE == Debug)
+// For gdb debug help...
+// Serialize a ngraph function into a file
+Status gdb_serialize_ngfunc(const char*,
```

Comment:
Prefix helpers with, say, `dbg_` instead of `gdb_` since these debug helpers aren't really specific to gdb per se.

Perhaps
`dbg_print_ngraph_function`
`dbg_serialize_ngraph_function`
so on and so forth.

Reply:
gdb branding will make it very clear that these are meant for debugger support, whereas, dbg_xxx could misguide future maintainers as useful debug calls to be used within code - which they can, but we are not compiling them  for non-debug-build. we can probably call these debugger_xxx(...), but that would lengthen the names a bit. Comments?